Language/Haskell/GhcMod/Browse.hs

```diff
@@ -26,12 +28,17 @@ import Prelude
 ----------------------------------------------------------------
 
 -- | Getting functions, classes, etc from a module.
---   If 'detailed' is 'True', their types are also obtained.
---   If 'operators' is 'True', operators are also returned.
 browse :: forall m. IOish m
        => String -- ^ A module name. (e.g. \"Data.List\", "base:Prelude")
        -> GhcModT m String
-browse pkgmdl = do
+browse = browseWith defaultBrowseOpts
```

Comment:
Just passing in the default options seems semantically wrong here, a library user could have modified the global options from the defaults and then called `browse` this wont be picked up now.

Just break the API ;)